pkg/network/protocols/http2/protocol.go

```diff
@@ -343,6 +353,47 @@ func (p *protocol) GetStats() *protocols.ProtocolStats {
 	}
 }
 
+// UpdateKernelTelemetry should be moved to the HTTP/2 part as well
+func (p *protocol) UpdateKernelTelemetry(mgr *manager.Manager) {
```

Comment:
please run load test and show the impact of the new code
I'm concerned with cpu-intensive code

Reply:
did you run it?

Reply:
@amitslavin gentle bump here
if you ran load test, then please add the results to the PR description